src/citrine/resources/delete.py

```diff
@@ -14,9 +15,10 @@
 
 def _async_gemd_batch_delete(
         id_list: List[Union[LinkByUID, UUID, str, BaseEntity]],
-        project_id: UUID,
         session: Session,
-        dataset_id: Optional[UUID],
+        project_id: Optional[UUID] = None,
+        team_id: Optional[UUID] = None,
+        dataset_id: Optional[UUID] = None,
```

Comment:
Why are these arguments getting re-ordered?  This is a breaking change, albeit to a private interface.  Why isn't there a `*, ` to block positional arguments?  `session` as positional argument doesn't really make sense.

Reply:
That looks like a mistake which should be reverted. Good catch.